src/components/index.js

```diff
@@ -1,4 +1,22 @@
-export { default as Text } from './Text'
-export { default as Button } from './Button'
+export {
+  Text,
+  Title,
+  Subtitle,
+  Header,
+  Subheader,
+  Description,
+  Body,
+  Date
+} from './texts'
+
+export {
+  Button,
+  GrayButton,
+  PrimaryButton,
+  AlphaButton,
+  BlackButton,
+  ImageButton
+} from './buttons'
```

Comment:
🤔 I don't like these `index.js` files that only export other things...
- First, you could've written this as `export * from './texts` (same for buttons).
- Second, these index files are another thing to keep maintained. It's easier to just import the thing you need from the appropriate path: `import GrayButton from '~/components/buttons/GrayButton'`